packages/forklift-console-plugin/src/modules/Overview/views/overview/tabs/Details/cards/ChartsCard.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from 'react';
+import { DateTime, Interval } from 'luxon';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { MigrationModelGroupVersionKind, V1beta1Migration } from '@kubev2v/types';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { Chart, ChartAxis, ChartBar, ChartGroup, ChartTooltip } from '@patternfly/react-charts';
+import {
+  Card,
+  CardActions,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  Dropdown,
+  DropdownItem,
+  Flex,
+  KebabToggle,
+} from '@patternfly/react-core';
+import chart_color_blue_200 from '@patternfly/react-tokens/dist/esm/chart_color_blue_200';
+import chart_color_green_400 from '@patternfly/react-tokens/dist/esm/chart_color_green_400';
+import chart_color_red_100 from '@patternfly/react-tokens/dist/esm/chart_color_red_100';
+
+import { MigrationsCardProps } from './MigrationsCard';
+
+interface MigrationDataPoint {
+  dateLabel: string;
+  value: number;
+}
+
+const toStarted = (m: V1beta1Migration): string => m.status.started;
+const toFinished = (m: V1beta1Migration): string => m.status.completed;
```

Comment:
this two methods can be one that return started time if not finished and finish time if finised (success or fail)

then we can remove the need to give the toDataPoint a method to get the timestamp

Reply:
The method are used also for differentiating finished migrations from others: a migration without valid `completed` timestamp will be filtered out at next step in the pipeline.